composer.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "wp-cli/sample-plugin",
+    "description": "Example plugin, generated using `wp scaffold plugin sample-plugin`",
+    "type": "wordpress-plugin",
+    "scripts": {
+        "lint:php:config": "phpcs --config-set installed_paths vendor/wp-coding-standards/wpcs",
+        "lint:php": "phpcs --standard=phpcs.ruleset.xml $(find src tests -name '*.php')",
+        "lint:php:fix": "phpcs --standard=phpcs.ruleset.xml $(find src tests -name '*.php')"
+    },
+    "require": {
+        "php": ">=5.3",
+        "composer/installers": "^1.3",
+        "wp-coding-standards/wpcs": "*"
+    }
+}
```

Comment:
Purposely omitted `license` field. Includes basic setup for `type="wordpress-plugin"` for easy plugin installation via composer. Adds scripts to configure, lint and fix PHP code. Commands are namespaced to allow planned introduction of `lint:js:*` scripts.